.github/CODEOWNERS

```diff
@@ -3,3 +3,4 @@ packages/core/src/codewhisperer/ @aws/codewhisperer-team
 packages/core/src/amazonqFeatureDev/ @aws/earlybird
 packages/core/src/codewhispererChat/ @aws/dexp
 packages/core/src/amazonq/ @aws/dexp
+packages/core/src/accessanalyzer/ @aws/access-analyzer
```

Comment:
@kevluu-aws please consider moving your code to `packages/toolkit/src/accessanalyzer` . That is where Toolkit-only code should live. /core/ is for code that may be useful for both Toolkit and Amazon Q.

Reply:
I will move it to the Toolkit folder in a future PR.